routes/rooms.go

```diff
@@ -56,6 +74,77 @@ func GetEmergencyRoomById(ctx *gin.Context) {
 	ctx.JSON(http.StatusOK, resp)
 }
 
+// GetEmergencyRooms godoc
+// @Summary    get all emergency rooms
+// @Tags       emergency-rooms
+// @Produce    json
+// @Param      page                            query       uint                    false   "0-indexed page number, 0 is assumed when omitted"
+// @Success    200                             {object}    GetMultipleERsResponse
```

Comment:
the GetMultipleERsResponse struct is now at the bottom of the file, it should be right before this function